convert_gvf_to_vcf/convertGVFtoVCF.py

```diff
@@ -142,18 +144,48 @@ def read_info_attributes(info_attributes_file):
             attribute_dict[key] = attribute_tokens
     return attribute_dict
 
+def read_sequence_ontology_symbolic_allele(so_symbolic_allele_file):
```

Comment:
There seems to be a common aspect between all the readers where we read a csv file and one of the column is a unique key (usually the first column) and the value is a list of all the column or a formatted string. There is some potential refactoring that could help with the readability of the code and remove some duplication. 
It would be good to keep that in mind in future improvement.